configs/focus_android_events_schemas.json

```diff
@@ -4,6 +4,640 @@
+                    "name": "Autocomplete URL tip tapped",
+                    "schema": {
+                        "$schema": "http://json-schema.org/schema#",
+                        "properties": {
+                            "category": {
+                                "enum": [
+                                    "action"
+                                ],
+                                "type": "string"
+                            },
+                            "method": {
+                                "enum": [
+                                    "click"
+                                ],
+                                "type": "string"
+                            },
+                            "object": {
+                                "enum": [
+                                    "tip"
+                                ],
+                                "type": "string"
+                            },
+                            "value": {
+                                "enum": [
+                                    "add_autocomplete_url_tip"
+                                ],
+                                "type": "string"
+                            },
+                            "timestamp": {
+                                "minimum": 0,
+                                "type": "number"
+                            }
+                        },
+                        "required": [
+                            "category",
+                            "timestamp",
+                            "object",
+                            "method",
+                            "value"
+                        ],
+                        "type": "object"
+                    }
+                },
+                {
+                    "amplitudeProperties": {},
+                    "description": "",
+                    "name": "Homescreen tips enabled/disabled",
+                    "schema": {
+                        "$schema": "http://json-schema.org/schema#",
+                        "properties": {
+                            "category": {
+                                "enum": [
+                                    "action"
+                                ],
+                                "type": "string"
+                            },
+                            "method": {
+                                "enum": [
+                                    "click"
+                                ],
+                                "type": "string"
+                            },
+                            "object": {
+                                "enum": [
+                                    "tip"
+                                ],
+                                "type": "string"
+                            },
+                            "value": {
+                                "enum": [
+                                    "add_to_homescreen_tip"
```

Comment:
This looks like a duplicate from L310 but a different title

Reply:
...How in the world did you notice that?